common/src/main/java/com/aliyun/rss/common/network/protocol/PushData.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.aliyun.rss.common.network.protocol;
+
+import com.google.common.base.Objects;
+import io.netty.buffer.ByteBuf;
+
+import com.aliyun.rss.common.network.buffer.ManagedBuffer;
+import com.aliyun.rss.common.network.buffer.NettyManagedBuffer;
+
+public final class PushData extends AbstractMessage implements RequestMessage {
+  public long requestId;
+
+  public int epoch;
+
+  // 0 for master, 1 for slave, see PartitionLocation.Mode
+  public final byte mode;
+
+  public final String shuffleKey;
+  public final String partitionUniqueId;
+
```

Comment:
Flink need some more fields for pushing data, should we create a new message type which extends this or we add these fields in this message?